scripts/test

```diff
@@ -26,6 +26,14 @@ then
             find ./scripts -type f -print0 | xargs -0 -r shellcheck
         fi
 
+        echo "Verifying that Airflow containers can run the batch jar"
+        docker-compose \
+            -f "${DIR}/../docker-compose.airflow.yml" \
+            run --rm \
+            airflow-worker \
+            bash -c \
+            "java -cp /opt/raster-foundry/jars/rf-batch.jar com.azavea.rf.batch.Main healthcheck"
+
```

Comment:
My guess is this leaving a container up and is what is causing this to fail in CI